core/src/components/alert/alert-interface.ts

```diff
@@ -13,6 +13,7 @@ export interface AlertOptions {
   backdropDismiss?: boolean;
   translucent?: boolean;
   animated?: boolean;
+  additionalAttributes?: AlertAttributes;
```

Comment:
Thoughts on calling this `htmlAttributes` similar to how we call it `cssClass`?

Reply:
Also I think JSX uses this same naming convention for attributes: https://github.com/ionic-team/ionic-framework/blob/7315e0157b917d2e3586c64f8082026827812943/core/src/components/radio/radio.tsx#L165